src/taoensso/timbre/appenders/3rd_party/udp_socket.clj

```diff
@@ -0,0 +1,20 @@
+(ns taoensso.timbre.appenders.3rd-party.udp-socket
+  {:author "Leo Zovic (@inaimathi)"}
+  (:import [java.net DatagramSocket DatagramPacket InetSocketAddress]))
+
+(defn udp-appender
+  [host port]
```

Comment:
Would request a docstring, and config as map `opts` for better flexibility in future if additional options are added, thanks!

Reply:
I'm assuming I can use the basic template from your `logstash` appender.

Reply:
Thanks Leo! For future, there's an [example appender](https://github.com/ptaoussanis/timbre/blob/master/src/taoensso/timbre/appenders/example.clj) that you can refer to.